definitions.yaml

```diff
@@ -221,6 +221,20 @@ definitions:
           size_gb:
             type: number
             description: RAM size in GB. Can be an integer or float.
+      scaling:
+        type: object
+        description: |
+          Attributes specific to cluster node scaling. To have full control of
+          cluster scaling, min and max can be set to the same value.
+        properties:
+          min:
+            type: integer
+            description: |
+              The minimum number of cluster nodes in this node pool
+          max:
+            type: integer
+            description: |
+              The maximum number of cluster nodes in this node pool
```

Comment:
I thought a bit about the API and what to do here for cluster creation, especially having node pools in mind. The API is already quite aligned with that I think. We have the `workers` list which contain all the node settings. Later we might just rename this to `nodes` or whatever. So now I just added the scaling section. Naming is up for discussion. Is this good enough for cluster creation?

Reply:
So do I understand this correctly that now `scaling: {min/max}` parameter is property of a single worker node and we use it like instance type etc. right now? This is definitely unintuitive to me (I'd rather have it in one step higher level) but as you write, this aligns with what we currently have. I don't care so much about this right now though, when we are moving away from these with node pools.

Reply:
Why should the `scaling` be defined per node? This doesn't make sense. I know the worker is already wrong in general as we only use the first worker configuration to create a pool of nodes.

Since we know already that this is gonna change (and fixed) with node pools and cluster api I am fine with keeping the worker list as is. But the `scaling` should go into the `V4ClusterDetailsResponse` right next to eg the availability zones.

Reply:
My thinking was that we use the workers list here as node pool list in the future and that you want to define the scaling settings per pool.